pkg/cmd/serviceaccount/create/create.go

```diff
@@ -140,8 +141,13 @@ func runCreate(opts *options) error {
 		defer httpRes.Body.Close()
 	}
 
-	if err != nil {
-		return err
+	if apiErr := errorutils.GetAPIError(err); apiErr != nil {
+		switch apiErr.GetError() {
+		case "service_account_limit_exceeded":
```

Comment:
I think we have Enums defined in SDK for those

Reply:
I tried that first. The enums are defined in a different manner in the openAPI [file](https://github.com/redhat-developer/app-services-sdk-go/blob/main/.openapi/service-accounts.yaml#L346), more like accepted options.